definitions/multi_product.toml

```diff
@@ -39,6 +39,286 @@ category = "KPI"
 owner = ["bochocki@mozilla.com", "firefox-kpi@mozilla.com"]
 deprecated = false
 
+##### search revenue forecasting metrics
+
+[metrics.search_forecasting_daily_active_users]
+data_source = "search_revenue_levers_daily"
+select_expression = ''{{agg_sum("dau")}}''
+friendly_name = "Daily Active Users"
+description = """
+    Counts the number of daily active users (DAU) for search revenue forecasting purposes. 
+"""
+category = "search"
+type = "scalar"
+
+[metrics.search_forecasting_daily_active_users_w_google_default]
+data_source = "search_revenue_levers_daily"
+select_expression = ''{{agg_sum("dau_w_engine_as_default")}}''
+friendly_name = "Daily Active Users with Google as Default"
+description = """
+    Counts the number of daily active users (DAU) with Google as default search engine for search revenue forecasting purposes. 
+"""
+category = "search"
+type = "scalar"
+
+[metrics.search_forecasting_daily_active_searchers_w_google_default]
+data_source = "search_revenue_levers_daily"
+select_expression = '{{agg_sum("dau_engaged_w_sap")}}'
+friendly_name = "Daily Active Users who Search with Google as Default"
+description = """
+    Counts the number of daily active users (DAU) with Google as default search engine who also conduct a search for search revenue forecasting purposes. 
+"""
+category = "search"
+type = "scalar"
+
+# note: this metric needs to be counted daily then aggregated monthly
+[metrics.search_forecasting_daily_active_searchers_w_google_mobile]
+data_source = "mobile_search_clients_engines_sources_daily"
+from_expression = "WITH
+-- Grab Mobile Eligible DAU
```

Comment:
Just spoke to Sydney about this -- there's a minor change in this query, where we have to use `search_engine` instead of `default_search_engine` because of a bug that's in the process of being fixed. We are removing this from this PR, though, and I'll add this query locally when I run the mobile forecasts.